test/ctr.bats

```diff
@@ -51,10 +50,9 @@ function wait_until_exit() {
 @test "ctr termination reason Error" {
 	start_crio
 	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
-
-	python -c 'import json,sys;obj=json.load(sys.stdin);obj["command"] = ["false"]; json.dump(obj, sys.stdout)' \
-		< "$TESTDATA"/container_config.json > "$TESTDIR"/container_config_error.json
-	ctr_id=$(crictl create "$pod_id" "$TESTDIR"/container_config_error.json "$TESTDATA"/sandbox_config.json)
+	jq '	  .command = ["false"]' \
```

Comment:
is the whitespace between ' and .command needed for the jq command to work?

Reply:
No, but in some cases there are multiple elements to be set, so I use this to unify formatting and be shfmt-compatible at the same time.

Reply:
Depending on github settings, it might look weird in github, but it looks fine in the terminal.

This is what I mean
![Screenshot from 2020-10-13 11-48-05](https://user-images.githubusercontent.com/4522509/95902794-24bc6700-0d4a-11eb-92ca-ec1b792e9f72.png)

I am using the same formatting everywhere so that more lines can be added later.